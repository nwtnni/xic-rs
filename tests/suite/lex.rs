#[test_generator::test_resources("tests/lex/*.xi")]
pub fn lex(path: &str) {
    let tokens = super::lex(path);
    insta::assert_display_snapshot!(path, tokens);
}
