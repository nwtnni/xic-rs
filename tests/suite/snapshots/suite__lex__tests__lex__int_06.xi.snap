---
source: tests/suite/lex.rs
expression: tokens
---
1:1 -
1:2 integer 9223372036854775808

